stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
The problem with doing that "for the present" is that you break user applications if you fix it later, and then you never fix it to avoid breaking user applications, and you're stuck with it.